src/graphql/types/object/wallet-contact.ts

```diff
@@ -43,7 +41,7 @@ const UserContact = new GT.Object({
           throw account
         }
 
-        const transactions = await Wallets.getAccountTransactionsForContact({
+        const transactions = await getAccountTransactionsForContact({
```

Comment:
Small nit, why not have this imported as `Accounts.getAccountTransactionsForContact` instead?

Reply:
no particular reasons